trinity/__init__.py

```diff
@@ -13,7 +13,7 @@
 # This is to ensure we call setup_trace_logging() before anything else.
 import eth as _eth_module  # noqa: F401
 
-if sys.platform in {'darwin', 'linux'}:
+if sys.platform in {'darwin', 'linux'} or 'freebsd' in sys.platform:
```

Comment:
This is nearly complex enough now that it's worth pulling into a separate function.

Also `sys.platform.startswith('freebsd')` is a bit stricter, which we would prefer when possible.

Reply:
Thanks @carver! I've changed code as per your suggestions. Now it fails somewhere in CircleCI, I believe it need to be simply restarted. I'll try to find something about using FreeBSD in CircleCI.